build/publish/PublishNupkgToTransportFeed.targets

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ItemsToPush Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <RelativePath>packages</RelativePath>
+  </PropertyGroup>
+
+  <Target Name="PublishNupkgToTransportFeed" Condition=" '$(PUBLISH_NUPKG_TO_TRANSPORT_FEED)' == 'true' AND '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
+    <Error Condition="'$(TransportFeedContainerName)' == ''" Text="Missing property TransportFeedContainerName." />
+    <Error Condition="'$(TransportFeedCloudDropAccountName)' == ''" Text="Missing property TransportFeedCloudDropAccountName." />
+    <Error Condition="'$(TransportFeedCloudDropAccessToken)' == ''" Text="Missing property TransportFeedCloudDropAccessToken." />
+
+    <Message Text="Publish to $(TransportFeedContainerName) started" />
+    <PushToBlobFeed AccountKey="$(TransportFeedCloudDropAccessToken)"
+                AccountName="$(TransportFeedCloudDropAccountName)"
```

Comment:
These parameters have been updated in the latest version of the Feed Package.

Reply:
There is just one version in the feed: https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/microsoft.dotnet.build.tasks.feed/1.0.0-prerelease-01929-02/